public/js/src/module/encryptor.js

```diff
@@ -25,6 +34,49 @@ function isSupported() {
         new Uint8Array( 8 ).length === 8;
 }
 
+const isEncryptionEnabledSymbol = Symbol( 'isEncryptionEnabled' );
+
+/**
+ * @param {Survey} survey
+ * @return {boolean}
+ */
+function isEncryptionEnabled( survey ) {
+    return Boolean( survey[isEncryptionEnabledSymbol] );
+}
+
+function serializeEncryptedSurvey( survey ) {
+    if ( isEncryptionEnabled( survey ) ) {
+        return Object.assign( {}, survey, {
+            isEncryptionEnabled: true,
+        } );
```

Comment:
In general I tend to avoid mutating function parameters unless there's a compelling reason to do so, as it tends to make it easier to reason about state/data flow.